src/modules/facility-investor/dto/create-facility-investor-request.dto.ts

```diff
@@ -0,0 +1,68 @@
+import { PROPERTIES } from '@ukef/constants';
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedNumberApiProperty } from '@ukef/decorators/validated-number-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+
+export type CreateFacilityInvestorRequest = CreateFacilityInvestorRequestItem[];
+
+export class CreateFacilityInvestorRequestItem {
+  // TODO APIM-93: Should we validate that it is only digits?
+  @ValidatedStringApiProperty({
+    description: 'The identifier of the facility to create the investor for.',
+    example: '0000000001',
+    minLength: 10,
+    maxLength: 10,
+  })
+  readonly facilityIdentifier: string;
+
+  @ValidatedDateOnlyApiProperty({
+    description: `The date from which this limit is effective.`,
+  })
+  readonly effectiveDate: DateOnlyString;
+
+  @ValidatedDateOnlyApiProperty({
+    description: `The date on which this limit will expire.`,
+  })
+  readonly guaranteeExpiryDate: DateOnlyString;
+
+  @ValidatedStringApiProperty({
+    description: 'The code of the currency for this investor, Currency in the Currency Definition Table.',
```

Comment:
Here it refers to `currency definition table`, it is a enum of all the valid currencies which can be acccepted?
If no then should we have `Enum` with all the valid currencies ISO codes only?

Reply:
Audrius pointed me to the `GET /Lists/Currency` endpoint in ACBS, which is the list of allowed currencies. These currencies can be added or removed in ACBS so we can't write an enum for them (Mulesoft doesn't limit them either)